HOWTO_RELEASE

```diff
@@ -0,0 +1,22 @@
+# Preparing the changelog
+
+Looks like gitchangelog is not as nice as thought before,
+however, you can generate the changelog by doing this:
+
+$ gitchangelog 0.2.3..HEAD 
```

Comment:
Just a suggestion - can we make the _FROM_ version a bit more generic - e.g. `0.x.x` instead of fixing it to 0.2.3? Just to force me to think in case I decide to copy-paste?